discord-bot/bot/api_client.py

```diff
@@ -89,23 +91,22 @@ async def fetch_random_task(
         logger.debug(f"Fetching random for user {user}")
         return await self.fetch_task(protocol_schema.TaskRequestType.random, user, collective)
 
-    async def ack_task(self, task_id: str | UUID, message_id: str):
+    async def ack_task(self, task_id: str | UUID, message_id: str) -> None:
         """Send an ACK for a task to the backend."""
         logger.debug(f"ACK task {task_id} with post {message_id}")
         req = protocol_schema.TaskAck(message_id=message_id)
-        return await self.post(f"/api/v1/tasks/{task_id}/ack", data=req.dict())
```

Comment:
Currently `tasks/{task_id}/ack` returns an empty dict. Probably we can change that to an empty 200 OK response too (which NACK currently does).

Reply:
028c918